src/production/deploy/standalone.md

```diff
@@ -2,4 +2,74 @@
 title: Deploy Kuma in standalone mode
 ---
 
-@TODO
\ No newline at end of file
+This is the simplest deployment mode for {{ site.mesh_product_name }}, and the default one.
+
+* **Control plane**: There is one deployment of the control plane that can be scaled horizontally.
+* **Data plane proxies**: The data plane proxies connect to the control plane regardless of where they are deployed.
+* **Service Connectivity**: Every data plane proxy must be able to connect to every other data plane proxy regardless of where they are being deployed.
+
+This mode implies that we can deploy {{ site.mesh_product_name }} and its data plane proxies in a standalone networking topology mode so that the service connectivity from every data plane proxy can be established directly to every other data plane proxy.
+
+<center>
+<img src="/assets/images/docs/0.6.0/flat-diagram.png" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
+</center>
+
+Standalone mode is usually a great choice within the context of one zone (ie: within one Kubernetes cluster or one AWS VPC).
+
+## Limitations
+
+* All data plane proxies need to be able to communicate with every other dataplane proxy.
+* A standalone deployment cannot mix Universal and Kubernetes workloads.
+* A deployment can connect to only one Kubernetes cluster at once.
+
+If these limitations are problematic you should look at [Multi-zone deployments](/docs/{{ page.version }}/deployments/multi-zone).
+
+## Usage
```

Comment:
@Guaris This one should be everything from the "Usage" section onwards. /src/production/deployment-topologies/standalone will contain everything above the "Usage" section.

Reply:
got it